src/Microsoft.DotNet.XHarness.Tests.Runners/xUnit/TestCaseExtensions.cs

```diff
@@ -0,0 +1,68 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using Xunit.Abstractions;
+
+namespace Microsoft.DotNet.XHarness.Tests.Runners.Xunit
+{
+    /// <summary>
+    /// Useful extensions that make working with the ITestCase interface nicer within the runner.
+    /// </summary>
+    public static class TestCaseExtensions
+    {
+        /// <summary>
+        /// Returns if the test case does have traits.
+        /// </summary>
+        /// <param name="testCase">The test case under test.</param>
+        /// <returns>true if the test case has traits, false otherwise.</returns>
+        public static bool HasTraits(this ITestCase testCase) =>
+            testCase.Traits != null && testCase.Traits.Count > 0;
+
+        public static bool TryGetTrait(this ITestCase testCase,
+                                       string trait,
+                                       [NotNullWhen(true)] out List<string>? values,
+                                       StringComparison comparer = StringComparison.InvariantCultureIgnoreCase)
+        {
+            if (trait == null)
+            {
+                values = null;
+                return false;
+            }
+
+            // there is no guarantee that the dict created by xunit is case insensitive, therefor, trygetvalue might
```

Comment:
therefore, TryGetValue()...

Reply:
@MattGal GitHub (AzDO now as well actually) has these handy suggestions that could make fixing these faster ^^

```suggestion
            // there is no guarantee that the dict created by xunit is case insensitive, therefore, trygetvalue might
```

Reply:
![image](https://user-images.githubusercontent.com/7013027/80993875-20575e00-8e3c-11ea-8055-c482707bdb03.png)